app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/presenter/AuthenticationPresenter.kt

```diff
@@ -48,7 +58,7 @@ class AuthenticationPresenter @Inject constructor(
     /**
      * Return information about the currently signed-in user
      */
-    val currentUser: User?
+    var currentUser: User? = null
```

Comment:
why this modification ?

Reply:
yep doesn't look necessary

Reply:
I have to do like this because now when signing out I have  set   _currentUser to  null

Reply:
that makes this duplicative of `_currentUser`, which is asking for trouble at this point.